client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -290,6 +290,21 @@ const Editor = (props: EditorProps): JSX.Element => {
     }
   };
 
+  const setAccessibilityMode = () => {
+    const accessibilityMode = {
+      isAccessiblityModeOn: props.inAccessibilityMode
+    };
+
+    if (!store.get('accessibilityMode')) {
+      store.set('accessibilityMode', accessibilityMode);
+    }
+
+    /* eslint-disable */
+    const accesibility = store.get('accessibilityMode').isAccessibilityModeOn;
+
+    return accesibility;
+  };
```

Comment:
```suggestion
  const setAccessibilityMode = () => {
    const accessibilityMode = {
      isAccessibilityModeOn: props.inAccessibilityMode
    };
    type AccessibilityMode = typeof accessibilityMode;

    function accessibilityModeOrNull(mode: unknown): AccessibilityMode | null {
      const access = mode as AccessibilityMode;
      return access.isAccessibilityModeOn ? access : null;
    }
    const accessibility = accessibilityModeOrNull(store.get('accessibility'));
    if (!accessibility) {
      store.set('accessibilityMode', accessibilityMode);
    }
    return accessibility?.isAccessibilityModeOn ?? false;
  };
```

Reply:
This is probably a bit redundant, as you might as well:
```suggestion
  const setAccessibilityMode = () => {
    const accessibilityMode = {
      isAccessibilityModeOn: props.inAccessibilityMode
    };
    type AccessibilityMode = typeof accessibilityMode;

    const accessibility = store.get('accessibilityMode') as AccessibilityMode;
    if (!accessibility) {
      store.set('accessibilityMode', accessibilityMode);
    }
    return accessibility?.isAccessibilityModeOn ?? false;
  };
```